mgmtfn/k8splugin/README.md

```diff
@@ -0,0 +1,311 @@
+# Contiv Networking for Kubernetes
+
+Contiv is integrated with Kubernetes via a CNI plugin. This integration enables you
+to set up and manage inter-pod connectivity in a Kubernetes cluster using Contiv 
+policy model.
+
+## Getting Started
+
+This step-by-step procedure will guide you through a minimal experience of creating
+a Kubernetes cluster with Contiv networking and applying policy between pods.
+
+#### Step 1: Clone and build kubernetes
+
+```
+$ mkdir ~/go/src/github.com/k8s
+$ cd ~/go/src/github.com/k8s
+$ git clone https://github.com/kubernetes/kubernetes
+$ cd kubernetes; make
+```
+
+Note: If the build fails, please retry. It will usually build the binaries on 2nd attempt
+
+#### Step 2: Clone contrib repo
+
+```
+$ cd ~/go/src/github.com/k8s
+$ git clone https://github.com/jojimt/contrib -b contiv
```

Comment:
We should open an issue to track this - assuming upstreaming can take some time (reviews from k8s team, and making in official master, etc.)